pkg/apiserver/config/config.go

```diff
@@ -207,6 +210,11 @@ func (conf *Config) ToMap() map[string]bool {
 			}
 		}
 
+		if name == "metering" {
+			result[name] = conf.MeteringOptions.Enable
+			continue
+		}
+
```

Comment:
```suggestion
```

Reply:
Should add some code into `stripEmptyOptions`

Reply:
Got it